bukkit/src/main/java/host/bloom/ab/bukkit/BukkitLoginListener.java

```diff
@@ -16,11 +18,21 @@ public BukkitLoginListener(BukkitPlugin plugin) {
     @EventHandler
     public void onPlayerJoinEvent(PlayerJoinEvent e) {
         plugin.getManager().incrementConnectionCount();
+
+        Player player = e.getPlayer();
+
+        if (player.hasPermission("bab.admin.actionbar")) {
+            this.plugin.getManager().addSeer(player.getUniqueId());
```

Comment:
Just to clarify, would we not want to respect it if they specifically disable it with the command? This way they have to disable it each join

Reply:
Yeah, either that or we could just track if they manually set it via the command and not overwrite that

Reply:
Oh so like bab.admin.actionbar.autoenable?

Reply:
I'll use a permission for now, in a later date... I'll add some file based storage for it as non persistent cache likely won't be the best.

Reply:
Sounds good! We can follow up with that later